Formula/xdk.rb

```diff
@@ -0,0 +1,47 @@
+class Xdk < Formula
+  desc "Ecstasy Development Kit (XDK)"
+  homepage "https://github.com/xtclang/xvm/"
+  url "https://github.com/xtclang/xvm/releases/download/v0.4.2/xdk-0.4.2.tar.gz"
+  sha256 "b0b5217abf05de986e6211a7c8665b304fa89df1726749e4f1dd7eff1cbbe1fe"
+  license "Apache-2.0"
+
+  depends_on "openjdk"
+
+  def install
+    %w[doc examples javatools lib README.md].each do |child_name|
```

Comment:
Why install README to libexec?

Reply:
>Why install README to libexec?

The reason that I included it is that all of those files "go together". Furthermore, since I'm stripping out the universal binaries when I install (see prior discussion above), the README isn't strictly necessary at this point. I will resubmit with it removed from the list.

One other strange thing: Despite what the code says, it places the README in the directory above the libexec, and does not place it in the libexec.